installer/MumbleInstall.cs

```diff
@@ -23,5 +29,21 @@ public class MumbleInstall : Project {
 		this.ControlPanelInfo.ProductIcon = @"..\icons\mumble.ico";
 		this.ControlPanelInfo.UrlInfoAbout = "https://mumble.info";
 		this.Properties = new Property[] { allUsersProp };
+		this.AfterInstall += Mumble_AfterInstall;
+		this.ValidateBackgroundImage = false;
+	}
+	
+	public static void Mumble_AfterInstall(SetupEventArgs e) {
+		if(e.IsInstalling) {
+			if(!IsVcredistInstalled()) {
+				var vcredist_path = "https://aka.ms/vs/17/release/vc_redist.{this.Arch}.exe";
+				Process.Start(vcredist_path);
+			}
+		}
+	}
+	
+	static bool IsVcredistInstalled() {
+		using (var key = Registry.LocalMachine.OpenSubKey(@"SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{6F330B47-2577-43AD-1195-1861BA25889B}"))
```

Comment:
I wonder... is this UUID the same for all architectures?

I'm assuming it is and the path to the registry key is redirected based on the arch. Right?

Reply:
I could only find that the x64 redist includes ARM64:

https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170

I can look into if further, but I don't know how much support is written into or accounted for in the 32 bit space. As far as I knew, we were only worrying about that for the plugins and the overlay. Are newer C++ features being used for those for 32 bit apps?

Reply:
This is what we would be dealing with starting with the entry for 2015-2019. All other cases (and probably this one also) we can't really anticipate in every single situation of an installed vcredist using registry and probably not using a folder path:

https://stackoverflow.com/a/34209692

The registry is probably the most accurate way to determine if something is installed and this still seems to make that process more difficult than it has to be.

Reply:
It makes more sense at this point to just have the installer for vcredist run regardless of their install state